src/MetricsDeclarations.ts

```diff
@@ -0,0 +1,13 @@
+import { IMetricsComponent } from '@well-known-components/interfaces'
+import { validateMetricsDeclaration } from '@well-known-components/metrics'
+
+export const metricDeclarations = {
+  dcl_published_collection_forum_post_failed: {
+    help: 'Count failed published collection forum posts',
+    type: IMetricsComponent.CounterType,
+    labelNames: ['forum'],
```

Comment:
I think that we can remove the label here, right? We are not using it

Reply:
Done!